doc/source/parameters/cfd/boundary_conditions_multiphysics.rst

```diff
@@ -36,8 +36,9 @@ The default parameters of each are shown:
 
 * ``id`` is the number associated with the boundary condition. By default, Lethe assumes that the id is equivalent to the number of the bc.
 
-* ``type``: This is the type of boundary condition been imposed. At the moment, choices are:
-    * ``temperature`` (Dirichlet BC), to impose a given temperature ``value`` at the boundary.
+* ``type``: type of boundary condition being imposed. At the moment, choices are:
+    * ``none`` so that there is no heat transfer boundary condition (nothing happens),
```

Comment:
I would replace that by no flux or a similar thing, see above comment. It's a bit easier to understand, because it's not that nothing happens, it's that no flux is assembled at the boundary.